uberenv.py

```diff
@@ -1179,6 +1193,10 @@ def main():
 
     # Allow to end uberenv after spack is ready
     if opts["setup_only"]:
+
+        if opts["upstream"] is not None:
```

Comment:
This, as well as the one below, will fail if running vcpkg instead of Spack. Can you either add these two guards against windows or add an error check in the function that parse's arguments here

https://github.com/LLNL/uberenv/blob/0e95ce0130db47aa015e8ca466cf038b2d76adca/uberenv.py#L250